core/domain/feedback_jobs_continuous_test.py

```diff
@@ -153,10 +153,9 @@ def test_no_threads(self):
     def test_single_thread_single_exp(self):
         with self._get_swap_context():
             exp_id = 'eid'
-            thread_id = 'tid'
+            thread_id = exp_id + '.tid'
```

Comment:
No, don't do that. Just refer to thread_id; remove references to exp_id. thread_id should just work as an arbitrary string here. Ditto below.

We should also add tests somewhere to ensure that thread IDs are correctly generated for their specific contexts. (But this isn't the purpose of this test, I think.)

Reply:
Done. I think when we generalize we'll need to have more tests. At the moment, I guess we are fine.